spec/rails_helper.rb

```diff
@@ -135,7 +135,8 @@ def filling_in_ssn_without_dashes(args)
         intercom_access_token: "fake_access_token",
         secure_mode_secret_key: "a-fake-key-to-use-for-hashing",
         statefile_secure_mode_secret_key: "a-fake-key-to-use-for-hashing"
-      }
+      },
+      mixpanel_token: "fake_mixpanel_token"
```

Comment:
Rails.logger _did_ receive an error without this change, b/c the mixpanel service was not able to instantiate a `@tracker` object without a `mixpanel_token` 

```
    mixpanel_key = Rails.application.credentials.dig(:mixpanel_token)
    return if mixpanel_key.nil?

    @buffer = []
    @mutex = Mutex.new

    @tracker = # from here it instantiates a tracker depending on env
```